monai/networks/nets/swin_unetr.py

```diff
@@ -0,0 +1,1001 @@
+# Copyright 2020 - 2022 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+from functools import reduce
+from operator import mul
+from typing import Sequence, Tuple, Type, Union
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.utils.checkpoint as checkpoint
+from torch.nn import LayerNorm
+
+from monai.networks.blocks.dynunet_block import UnetOutBlock
+from monai.networks.blocks.mlp import MLPBlock as Mlp
+from monai.networks.blocks.unetr_block import UnetrBasicBlock, UnetrUpBlock
+from monai.networks.layers import Conv
+from monai.utils import ensure_tuple_rep, optional_import
+
+rearrange, _ = optional_import("einops", name="rearrange")
+
+
+class SwinUNETR(nn.Module):
```

Comment:
please add this to https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/nets/__init__.py

Reply:
Will be added.